configure

```diff
@@ -165,6 +165,10 @@ _taskbar=auto
 _updates=no
 _libunity=auto
 _dialogs=auto
+_tts=auto
+_linux_tts=no
+_windows_tts=no
+_macosx_tts=no
```

Comment:
Why do you need three defines here? You could just combine one define with the target platform